sessions/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "session-handling",
+  "version": "0.0.1",
```

Comment:
Thinking out loud - should we drop version, and add `private: "true"`?

Reply:
Yes, done.